articles/aks/operator-best-practices-cluster-security.md

```diff
@@ -53,6 +53,10 @@ For more information about Azure AD integration, Kubernetes RBAC, and Azure RBAC
 >
 > Add a network policy in all user namespaces to block pod egress to the metadata endpoint.
 
+> [!Note]
+> For Network Policy Cluster needs to be created with attribute `--network-policy azure`
```

Comment:
Suggest rewording to -

To implement Network Policy, include the attribute `--network-policy azure` when creating the AKS cluster. Use the following command to create the cluster.

Reply:
completed